README.md

```diff
@@ -144,3 +144,4 @@ chmod +x *.sh
 ./docker-build.sh
 ./docker-run.sh
 ```
+You may then connect to https://localhost:8443/cas/ and log in as casuser / Mellon
```

Comment:
This is generally only relevant for folks running 8443. It might be better to simply state the credentials and direct one to refer to logs for url access.